packages/queued-request-controller/src/QueuedRequestMiddleware.test.ts

```diff
@@ -37,8 +40,12 @@ const buildMocks = (
   messenger: ReturnType<typeof buildMessenger>,
   mocks: {
     getNetworkClientById?: NetworkController['getNetworkClientById'];
-    getProviderConfig?: NetworkControllerGetStateAction['handler'];
+    getProviderConfig?: () => ProviderConfig;
     addRequest?: ApprovalController['add'];
+    // since NetworkConfigurations is not exported, we get it this way. Todo: export the type or expose a getter on NetworkController
+    getNetworkConfigurations?: () => ReturnType<
```

Comment:
[nit] I feel like its slightly confusing to name this `getNetworkConfigurations` since this isn't a real method (yet?) should we not just mock the actual method we call NetworkController[getState]?

Reply:
I can see the confusion when you consider that this might become an actual method, but the reason I did it this way  was  just to minimize the test refactoring and to make it easy enough to mock out the 'getting of networkconfigurations' seperate  from the  provider  config (i.e. you can mock out one and not the other if you want)

Reply:
Ok I found you ..
And could possibly send you a request


On this quote of yours ..
Could you possibly trace my actions on a site ..
Can you find me ..
Can you trace my digital steps right now can you separate the 2 metamaskaccounts by the 2 different digital  key performances